app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
Python check keys by default, so it could be simplified. Also if we compare to None it's better to use is instead of '=='
```suggestion
        if "vaccine" not in vstr or vstr["vaccine"] is None:
```